.tours/codeql-tutorial.tour

```diff
@@ -4,20 +4,29 @@
   "isPrimary": true,
   "steps": [
     {
-      "description": "First let's talk about the CodeQL CLI.\n\nThe CodeQL extension uses the CodeQL CLI to compile and run queries. It automatically manages access to the executable of the CLI for you.\n\nWe've already set it up to use in this tutorial. To learn more about the CodeQL CLI, see [the docs](https://codeql.github.com/).",
+      "description": "First thing's first, let's get some setup out of the way.\n\nIn the next step you will be prompted to open the tutorial workspace and allow telemetry.\n\nOnce you've done this, please return to this step in the Code Tour so we can explore the CodeQL extension.\n\n Now please click 'Next (The CodeQL Extension)'.",
```

Comment:
Let's make this a bit more generic! (We don't know what the user's workspace looks like: they might have already loaded the correct workspace, clicked around to activate the extension, accepted telemetry etc) 

Rough suggestion— edit at will! 👇🏽 

(Also suggested minor tweaks to keep "code tour" and "extension" lowercase, for consistency with later steps and the repo's README)

```suggestion
      "description": "Welcome to the CodeQL tour!\n\nIn the next steps, we'll start exploring the CodeQL extension. If this is your first time activating the extension, you may be prompted to open the tutorial workspace and decide whether to allow telemetry.\n\nOnce you've done this, return to this step in the code tour so we can explore the CodeQL extension.\n\n Now please click 'Next (The CodeQL extension)'.",
```

Reply:
(Just saw Nora's comment above too, with further suggestions 🙇🏽‍♀️)